packages/wallets/src/features/cashier/flows/WalletTransfer/__tests__/WalletTransfer.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { useTransferBetweenAccounts } from '@deriv/api';
+import { render, screen } from '@testing-library/react';
+import WalletTransfer from '../WalletTransfer';
+
+jest.mock('../../../../../components', () => ({
+    ...jest.requireActual('../../../../../components'),
+    Loader: jest.fn(() => <div>Loading</div>),
+}));
+
+jest.mock('../../../modules', () => ({
+    ...jest.requireActual('../../../modules'),
+    TransferModule: jest.fn(() => <div>TransferModule</div>),
+}));
+
+jest.mock('../../../screens/TransferNotAvailable', () => ({
+    ...jest.requireActual('../../../screens/TransferNotAvailable'),
+    TransferNotAvailable: jest.fn(({ children }) => (
+        <>
+            <div>TransferNotAvailable</div>
+            <div>{children}</div>
+        </>
+    )),
+}));
+
+jest.mock('@deriv/api', () => ({
+    ...jest.requireActual('@deriv/api'),
+    useTransferBetweenAccounts: jest.fn(),
+}));
+
+const mockUseTransferBetweenAccounts = useTransferBetweenAccounts as jest.MockedFunction<
+    typeof useTransferBetweenAccounts
+>;
+
+describe('<WalletTransfer />', () => {
+    it('should show the loader if the API response has not yet arrived', () => {
```

Comment:
the componetn is also displaying loader when we receive no data, which IMO is bad behavriour in the component itself. So in a way, its good that there is no test case for component misbehaviour just to get coverage :D